### DISCUSSION TITLE
[no-systemd]Can you consider no-systemd distro?

### DISCUSSION THREAD

**Discussion by Obarun** [linuxdeepin/developer-center#2847] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi there, can you consider no-systemd distro?

Your misc/pam-module/deepin_auth.c force to use systemd libraries. I understand the fact that's easy to code but this is not portable.

Regards

**Comment by jouyouyun** [comment]
May consider, but not now.

**Comment by Obarun** [comment]
ok, i will wait, thanks for your attention

**Comment by Obarun** [comment]
Any chance this year to have consideration about no-systemd distro?

**Comment by jouyouyun** [comment]
Sorry，no plan in 2020.

**Comment by Obarun** [comment]
Any chance during this year(2021) to have consideration about no-systemd distro?

**Comment by Obarun** [comment]
So, maybe this year(2022) you will finally consider the world outside systemd?

**Reply by BLumia** [reply]
Currently still not since the main devs from the company is still working on other major stuff.

Luckly most of DDE apps weren't rely on systemd at all so systemd is not required to use DDE apps. But for the DE itself, there might need some other people from the community to work on it to make it usable under non-systemd environments. (p.s. the portability of DDE is pretty bad at this moment and we are working on addressing this issue first before start doing any other thing)

If you are interested in hacking on it, feel free to start a discussion thread if you got anything we can help.

**Comment by BLumia** [comment]
Some progress on DTK https://github.com/linuxdeepin/dtkcore/pull/253, made by @xzl01
